packages/server/graphql/types/AtlassianIntegration.ts

```diff
@@ -108,12 +108,7 @@ const AtlassianIntegration = new GraphQLObjectType<any, GQLContext>({
           standardError(err, {tags: {teamId, userId}, userId: viewerId})
           return connectionFromTasks([], 0, err)
         }
-        if (!accessToken) {
-          const err = new Error('Not integrated with Jira')
-          standardError(err, {tags: {teamId, userId}, userId: viewerId})
-          return connectionFromTasks([], 0, err)
-        }
-        const manager = new AtlassianServerManager(accessToken)
```

Comment:
Even though I've tested all the scenarios I thought of and it all works I still do not feel confident about that change. Technically, this field is not nullable, am I missing some scenario in which this condition would be needed?

Reply:
looks great! i think the chance of a missing accessToken was from the beforetimes when we didn't guarantee its freshness. now we do in all cases